src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+$grey: rgba(97, 96, 112, 1);
+$main_accent: rgba(6, 11, 53, 1);
+$blue: rgba(0, 172, 220, 1);
+$hover_color: rgba(243, 243, 243, 1);
+
+.card {
+  padding: 32px 0 16px;
+  border: 1px solid hsl(0, 0%, 95%);
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  &--width {
+    width: 200px;
+  }
+  &__img-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__content {
+    margin: 0 16px ;
+  }
+  &__description {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__product-code {
+    color: $grey;
+    font-size: 10px;
+    font-weight: 400;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__count {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    display: flex;
+    color: $grey;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__price-sum {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
i think will be good create separated file `mixins` move these lines there, and reuse them